doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2472 @@
+  col_vector.shape
+
+**Output:**
+
+::
+
+  (6, 1)
+
+You can also expand an array by inserting a new axis at a specified position with ``np.expand_dims``.
+
+For example, if you start with this array:
+
+**Input:**
+
+::
+
+  a = np.array([1, 2, 3, 4, 5, 6])
+  a.shape
+
+**Output:**
+
+::
+
+  (6,)
+
+You can use ``np.expand_dims`` to add an axis at index position 1 with:
+
+**Input:**
+
+::
+
+  b = np.expand_dims(a, axis=1)
+  b.shape
+
+**Output:**
+
+::
+
+  (6, 1)
+
+You can add an axis at index position 0 with:
+
+**Input:**
+
+::
+
+  c = np.expand_dims(a, axis=0)
+  c.shape
+
+**Output:**
+
+::
+
+  (1, 6)
+
+`Find more information about newaxis here <https://numpy.org/devdocs/reference/arrays.indexing.html#index-1>`_ and `expand_dims here <https://numpy.org/devdocs/reference/generated/numpy.expand_dims.html>`_.
+
+Indexing and slicing
+--------------------
+
+You can index and slice NumPy arrays in the same ways you can slice Python lists.
+
+**Input:**
+
+::
+
+    data = np.array([1,2,3])
+
+    print(data[0])
+    print(data[1])
+    print(data[0:2])
+    print(data[1:])
+    print(data[-2:])
+
+**Output:**
+
+::
+
+  1
+  2
+  [1 2]
+  [2 3]
+
+You can visualize it this way:
+
+.. image:: images/np_indexing.png
+
+
+You may want to take a section of your array or specific array elements to use in further analysis or additional operations. To do that, you'll need to subset, slice, and/or index your arrays. 
+
+If you want to select values from your array that fulfill certain conditions, it's straightforward with NumPy. 
+
+For example, if you start with this array:
+
+::
+
+  a = np.array([[1 , 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
+
+You can easily print all of the values in the array that are less than 5.
+
+**Input:**
+
+::
+
+  print(a[a<5])
+
+**Output:**
+
+::
+  
+  [1 2 3 4]
+
+You can also select, for example, numbers that are equal to or greater than 5, and use that condition to index an array.
+
+**Input:**
+
+::
+
+  five_up = (a >= 5)
+  print(a[five_up])
+
+**Output:**
+
+::
+
+  [ 5  6  7  8  9 10 11 12]
+
+You can select elements that are divisible by 2:
+
+**Input:**
+
+::
+
+  divisible_by_2 = a[a%2==0]
+  print(divisible_by_2)
+
+**Output:**
+
+::
+
+  [ 2  4  6  8 10 12]
+
+Or you can select elements that satisfy two conditions using the ``&`` and ``|`` operators:
+
+**Input:**
+
+::
+
+  c = a[(a > 2) & (a < 11)]
+  print(c)
+
+**Output:**
+
+::
+
+  [ 3  4  5  6  7  8  9 10]
+
+You can also make use of the logical operators **&** and **|** in order to return boolean values that specify whether or not the values in an array fulfill a certain condition. This can be useful with arrays that contain names or other categorical values.
+
+**Input:**
+
+::
+
+  five_up = (array > 5) | (array == 5)
+  print(five_up)
+
+**Output:**
+
+::
+
+  [[False False False False]
+   [ True  True  True  True]
+   [ True  True  True  True]] 
+
+You can also use ``np.where()`` to select elements or indices from an array. 
```

Comment:
Can we teach `np.nonzero` as the recommended spelling? I've seen `where` confuse people because it has two completely different meanings (the one and three argument versions). `nonzero` is identical to the one-argument version, and comes without this confusion.

Reply:
Will do!

Reply:
Done!